app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -25,41 +25,9 @@ keywords:
     - Grants
     - Financial Support
     - Undergraduate
-    - Troops to Teachers
-    - Army
-    - Air Force
-    - RAF
-    - Navy
-    - Military
 ---
```

Comment:
Line 3 (heading): Would it be worth making the h1 this clearer - eg `Bursaries and scholarships for postgraduate training`? (although this is very long!)

We should keep the title as is for SEO though

Reply:
Agree - @jenhadfield-dfe would you be able to help make this change until I get access to codespaces again pls? It's not letting me add a suggestion through github.